src/printer.js

```diff
@@ -59,41 +64,14 @@ function getNodeIndex(path) {
   return body.indexOf(path.getValue());
 }
 
-function genericPrint(path, options, print) {
-  const n = path.getValue();
-  if (!n) {
-    return "";
-  } else if (typeof n === "string") {
-    return n;
-  }
-  if (n.kind === "program" && n.children.length === 0) {
-    return concat(["<?php", hardline]);
-  }
-  const printed = printNode(path, options, print);
+function lineShouldEndWithSemicolon(path) {
```

Comment:
github is doing a terrible job displaying this diff ☹️

Reply:
@mgrip agree :disappointed: